src/ontology/ro-edit.owl

```diff
@@ -3715,12 +3731,12 @@ SubObjectPropertyOf(obo:RO_0002412 obo:RO_0002411)
 # Object Property: obo:RO_0002413 (provides input for)
 
 AnnotationAssertion(obo:IAO_0000115 obo:RO_0002413 "p provides input for q iff p is immediately causally upstream of q, and there exists some c such that p has_output c and q has_input c.")
-AnnotationAssertion(obo:IAO_0000116 obo:RO_0002413 "This is currently called 'directly provides input for' to be consistent with our terminology where we use 'direct' whenever two occurrents succeed one another directly. We may relabel this simply 'provides input for', as directness is implicit")
```

Comment:
Why was this comment deleted?

Reply:
I deleted the comment because the term label was changed to 'provides input for' (see https://github.com/oborel/obo-relations/issues/651), so the comment is not relevant anymore